daemon/service/service_nocgo.go

```diff
@@ -0,0 +1,9 @@
+// +build !cgo
+
+// Package service implements the daemon service for mediating access to the
+// storage backend.
+package service
+
+// A dummy implementation of Serve for the environment of Cgo being disabled.
+func Serve(sockpath, dbpath string) {
```

Comment:
Hmm, I am not sure what you mean. What is the problem you are trying to solve? Your current approach seems good enough to me.